technologies/css.md

```diff
@@ -727,6 +728,7 @@ But in the following example we would not use it:
 [Nature playbook(private)]: https://github.com/nature/playbook
 [bem]: https://github.com/springernature/frontend/practices/bem-css.md
 [a hybrid of OOCSS and Atomic CSS]: https://github.com/springernature/frontend/practices/mosaic-css.md
```

Comment:
Can you clarify what you mean? 😁 is there a page about atomic, or something else?

(that "hybrid of" link is really confusing as it exists as a reference but isn't actually used by anything, and i don't know enough about the CSS stack here to be able to accurately fix things)